screed/screedRecord.py

```
@@ -129,6 +112,18 @@ def __ne__(self, given):
         except AttributeError:
             raise TypeError("Cannot compare to given type: %s" % type(given))
 
+    def __lt__(self):
+        return NotImplementedError
```

Typically one throws exception classes instead of returning them.